baker/batchTagWithGpt.ts

```diff
@@ -0,0 +1,88 @@
+import * as db from "../db/db.js"
+import { Chart } from "../db/model/Chart.js"
+import yargs from "yargs"
+import { hideBin } from "yargs/helpers"
+
+interface BatchTagWithGptArgs {
+    debug?: boolean
+    limit?: number
+}
+
+export const batchTagWithGpt = async ({
+    debug,
+    limit,
+}: BatchTagWithGptArgs = {}) => {
+    await batchTagChartsWithGpt({ debug, limit })
+}
+
+const batchTagChartsWithGpt = async ({
+    debug,
+    limit,
+}: BatchTagWithGptArgs = {}) => {
+    // Get all charts that need tagging. These charts either have no tags, or
+    // are tagged with neither a topic nor "Unlisted"
```

Comment:
Might just be me, but I find this phrasing with "either", "neither", and "nor" confusing 🫠

Unless I'm misunderstanding the query, `Get all charts that aren't tagged with a topic tag or the "Unlisted" tag` is clearer to me.

Reply:
Yes, you're right. I just wanted to emphasize that we also need to get charts that have no tags at all, which a previous version of the query wrongly omitted.